aspnetcore/tutorials/signalr-blazor-webassembly/samples/3.x/BlazorSignalRApp/Client/Pages/Index.razor

```diff
@@ -52,4 +53,9 @@
 
     public bool IsConnected =>
         hubConnection.State == HubConnectionState.Connected;
+        
+    public void Dispose()
+    {
+        _ = _hubConnection.DisposeAsync();
```

Comment:
Oh whoops, this should be `hubConnection` not `_hubConnection`

Reply:
I'll patch it.